src/js/signin/verify-entry.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { Provider } from 'react-redux';
+
+import initReact from '../common/init-react';
+import createCommonStore from '../common/store';
+import reducer from '../login/reducers/login';
+
+import Main from './containers/Main';
+
+require('../common');  // Bring in the common javascript.
+require('../../sass/login.scss');
+
+const commonStore = createCommonStore(reducer);
+
+export default function createLoginWidget(store, rootElementId = 'login-root') {
+  function init() {
+    ReactDOM.render((
+      <Provider store={store}>
+        <Main verify/>
```

Comment:
Do we need to have separate signin and verify bundles? It seems like they could be the same bundle, but set this verify param (and skip createLoginWidget) based on the url.

It might be slightly bigger, but then it would be cached for the second request on the verify step.